core/src/main/scala/akka/persistence/r2dbc/internal/postgres/PostgresDurableStateDao.scala

```diff
@@ -93,6 +100,9 @@ private[r2dbc] class PostgresDurableStateDao(
     settings.connectionFactorySettings.poolSettings.closeCallsExceeding)(ec, system)
 
   private implicit val statePayloadCodec: PayloadCodec = settings.durableStatePayloadCodec
+  implicit val timestampCodec: TimestampCodec = settings.timestampCodec
+  private implicit val tagsCodec: TagsCodec = settings.tagsCodec
+  implicit val queryAdapter: QueryAdapter = settings.queryAdapter
```

Comment:
This and the `tiestampCodec` are just for internal access, right? `protect` or `private` em' to show intent